sirepo/package_data/template/radia/radia_export.py.jinja

```diff
@@ -14,7 +14,7 @@ _MODELS = (_MODEL_BOX, _MODEL_GROUP, _MODEL_GROUP_UND, _MODEL_OBJECT)
 radia_objs = []
 sirepo_objs = {{ objects }}
 
-{% include 'radia_tk.py' %}
+{% include 'radia_util.py' %}
```

Comment:
Does this actually work? It would seem a simple import would be good enough. radia_util.py imports sirepo.util so it seems that an import is required anyway.

Reply:
It does work, in concert with this in ```template/radia.py```:

```
    return template_common.render_jinja(SIM_TYPE, v, j_file, jinja_env=PKDict(loader=jinja2.PackageLoader('sirepo', 'template')))
```

This is basically a sop to Oleg but incomplete as you can see...